docs/repositories-configure/integrations/github-integration.md

```diff
@@ -42,17 +42,31 @@ Adds a report to your pull requests showing whether your pull requests and cover
 
 ![Pull request status check on GitHub](images/github-integration-pr-status.png)
 
-### Annotations
+### Issue annotations
 
 Adds annotations on the lines of the pull request where Codacy finds new issues. Codacy maps the severity of the issues reported by the tools to the severity levels of the annotations. To enable this option, you must enable **Status checks** first.
 
-![Pull request annotation on GitHub](images/github-integration-pr-annotation.png)
+![Issue annotation on GitHub](images/github-integration-pr-annotation.png)
 
-### Summary
+### Issue summaries
 
 Shows an overall view of the changes in the pull request, including new issues and metrics such as complexity and duplication. To enable this option, you must enable **Status checks** first.
 
-![Pull request summary on GitHub](images/github-integration-pr-summary.png)
+![Issue summary on GitHub](images/github-integration-pr-summary.png)
+
+### Coverage summaries
+
+Adds a pull request comment showing an overall view of the coverage metrics for the pull request, including details about the data that Codacy used to calculate the coverage variation and diff coverage metrics.
+
+When there are new coverage results, Codacy updates the last coverage summary comment if it's included in the last 5 comments of the pull request. Otherwise, Codacy creates a new comment.
+
+!!! important
+    **To get coverage summaries** you must also [add coverage to your repository](../../coverage-reporter/index.md).
+
+![Coverage summary on GitHub](images/github-integration-coverage-summary.png)
```

Comment:
Do you think that it would be worth it to show all possible versions of this notification in docs, or would that be too much?

Reply:
That would definitely be too much - it would take too much space on the page in exchange for very little value.

I tend to prefer a more minimalist approach to docs, which works well when the UX is good and users can pretty much figure things out on their own. We can always add more details later if we get feedback suggesting that something's not clear or missing. :+1: